README.md

```diff
@@ -1,2 +1,43 @@
-# trustymail
-Scan domains and return data based on trustworthy email best practices
+## Trustworthy Mail  :lock:
+`TrustyMail` is a tool that evaluates SPF/DMARC DNS record configurations to recommend best practices for the records. It saves its results to a CSV.
+
+#### Usage and examples
+
+```bash
+python cli [options] INPUT
```

Comment:
Will do. I used the python run examples as I had not set up the module pip stuff yet. I will do that so we can have it installed as a module.